src/utils/getContentLength.js

```diff
@@ -0,0 +1,18 @@
+export default function getContentLength(content) {
+  let len;
```

Comment:
Don't use undefined as an initial value.  
Need to validate here(https://github.com/webpack/webpack-dev-middleware/pull/617/files#diff-f74cf7ea3fda676a7bafaf4bd8714ea4R83) or set 0.